papers/templates/papers/index.html

```diff
@@ -24,8 +24,9 @@ <h2>{% trans "Welcome to dissemin" %}</h2>
 
     <p class="startButton">
     {% blocktrans trimmed %}
-    Dissemin detects papers behind pay-walls and invites their authors to
-    upload them in one click to an open repository.
+    With Dissemin, researchers can identify which works of theirs are behind
+    paywalls, and archive such papers in an open repository, in few clicks,
+    with an appropriate copyright statement.

```

Comment:
Maybe we can do something like "...in an open repository with compliance to the repository guidelines, in a few clicks"

Maybe a bit gambling, but aside from Zenodo the administration does whatever check they think is necessary.